tests/func-tests/monitoring_test.go

```diff
@@ -63,6 +68,37 @@ var _ = Describe("[crit:high][vendor:cnv-qe@redhat.com][level:system]Monitoring"
 		prometheusRule = getPrometheusRule(ctx, restClient)
 
 		initialOperatorHealthMetricValue = getMetricValue(ctx, promClient, "kubevirt_hyperconverged_operator_health_status")
+
+		By("adding temporary route")
```

Comment:
Please (conditionally) do it only once for the whole test suite in the `BeforeSuite` node with a `DeferCleanup(...)` to clean it up at the end.

Reply:
done